.pre-commit-hooks.yaml

```
@@ -14,3 +14,11 @@
   language: system
   types:
     - lua
+
+- id: stylua-release
+  name: StyLua Release
```

Should probably be aligned with the other names
```suggestion
  name: StyLua (release)
```

Also is release the right name here? Maybe github instead? I'm not really sure myself

---

Hmm. Yea, I think `(Github)` or `(Github Release)` makes sense to give a better idea of where it's actually coming from.